grpc-web-client/src/lib.rs

```diff
@@ -1,4 +1,15 @@
 mod call;
+use cfg_if::cfg_if;
+
+cfg_if! {
+    if #[cfg(feature = "web-worker")] {
+        mod worker;
+        use crate::worker as request;
+    } else {
+        mod browser;
+        use crate::browser as request;
+    }
+}
```

Comment:
If you didn't want to use a library you could do this:

```suggestion
#[cfg(feature = "web-worker")]
mod worker;
#[cfg(feature = "web-worker")]
use crate::worker as request;

#[cfg(not(feature = "web-worker"))]
mod browser;
#[cfg(not(feature = "web-worker"))]
use crate::browser as request;
```

Reply:
True and that's how I had it before but I felt the `cfg_if!` option made it clearer.